amaranth/ml/train.py

```diff
@@ -17,12 +18,16 @@
 FDC_DATA_DIR = '../../data/fdc/'  # Data set directory
 MODEL_IMG_DIR = '../../docs/img/'  # Model image directory
 RESOURCES_DIR = '../resources/'  # Project resources directory
-
+CHROME_EXT_DIR = 'amaranth-chrome-ext/'  # Chrome extension directory
 # Fraction of data that should be used for training, validation, and testing.
 # Should all sum to 1.0.
 TRAIN_FRAC = 0.6
 VALIDATION_FRAC = 0.2
 TEST_FRAC = 0.2
+# Times a token needs to appear to be in model's vocab
+MIN_TOKEN_APPEARANCE = 0
```

Comment:
to be clear, you are adding every token? how do you deal with OOV tokens then?

Reply:
Shoot, I set it to zero during debugging. It should be 3, I'll check that out!

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/35/commits/f5ffbf7e2d8fa72bffdb29d0e587e6cc3fb6115c